go.mod

```diff
@@ -1,35 +1,33 @@
 module github.com/operator-framework/catalogd
 
-go 1.21
-
-toolchain go1.21.7
+go 1.22.0
```

Comment:
It seems to override ART change which set:

```go.mod
go 1.21

toolchain go1.21.7
```

I think the next time we merge an ART PR we will have a broken sync again due to a conflict here.

Reply:
This complaint has been pushed up to ART as a known issue. They did this on purpose, and are reconsidering. @joelanford